tests/dialects/test_snowflake.py

```diff
@@ -1548,6 +1548,29 @@ def test_show(self):
         self.assertTrue(isinstance(users_exp, exp.Show))
         self.assertEqual(users_exp.this, "USERS")
 
+    def test_storage_integration(self):
+        ast = parse_one(
+            """CREATE STORAGE INTEGRATION s3_int
+TYPE=EXTERNAL_STAGE
+STORAGE_PROVIDER='S3'
+STORAGE_AWS_ROLE_ARN='arn:aws:iam::001234567890:role/myrole'
+ENABLED=TRUE
+STORAGE_ALLOWED_LOCATIONS=('s3://mybucket1/path1/', 's3://mybucket2/path2/')""",
```

Comment:
Some of these properties are mandatory. I wasn't sure if I should be creating a Properties subclass for them (or something else)?

Reply:
Should be good if this is already parseable into a `Property` due to them all following this key = value syntax. We usually define new properties if there's custom / unrecognized syntax for them, so we can also map them back to that syntax later on.